### PR TITLE
 base-files: add reboot only button handler

### PR DESCRIPTION
**Pull request by alanswanson**
For devices such as BTHOMEHUBV5A with both reset and restart buttons, its easily accessible restart button has been assigned to KEY_POWER power script to poweroff preventing accidental (or malicious) factory resets by KEY_RESTART reset script. However an easily accessible button immediately powering off the device is also undesirable.

As KEY_RESTART is already used for reset script (and there's no KEY_REBOOT in Linux input events), use KEY_POWER2 for rebooting via new reboot script with 5 second seen delay.

Fixes FS#1965

### CODE REVIEW

File: package/kernel/button-hotplug/src/button-hotplug.c [hunk 1]
```diff
@@ -85,6 +85,7 @@ static struct bh_map button_map[] = {
 	BH_MAP(BTN_9,		"BTN_9"),
 	BH_MAP(KEY_RESTART,	"reset"),
 	BH_MAP(KEY_POWER,	"power"),
+	BH_MAP(KEY_POWER2,	"reboot"),
```
Comment (ynezz) [inline package/kernel/button-hotplug/src/button-hotplug.c]: This change doesn't correspond to the commit description (base-files package). It should go into separate commit.

File: package/kernel/gpio-button-hotplug/src/gpio-button-hotplug.c [hunk 2]
```diff
@@ -100,6 +100,7 @@ static struct bh_map button_map[] = {
 	BH_MAP(KEY_LIGHTS_TOGGLE,	"lights_toggle"),
 	BH_MAP(KEY_PHONE,		"phone"),
 	BH_MAP(KEY_POWER,		"power"),
+	BH_MAP(KEY_POWER2,		"reboot"),
```
Comment (ynezz) [inline package/kernel/gpio-button-hotplug/src/gpio-button-hotplug.c]: This change doesn't correspond to the commit description (base-files package). It should go into separate commit.

**Comment by ynezz** [comment]
@alanswanson Ping.

**Comment by alanswanson** [comment]
Sorry for the delay. Now split hotplug buttons changes into separate commits.

**Comment by ynezz** [comment]
Thanks, I've just pushed it to master.
